aemedge/libs/profiles/profiles.js

```diff
@@ -0,0 +1,30 @@
+import { loadCSS } from '../../scripts/aem.js';
+import { div } from '../../scripts/dom-builder.js';
+import Avatar from '../avatar/avatar.js';
+
+export default class Profiles {
```

Comment:
Instead of having `Profiles` as a reusable element we should have `Profile` like Card or Avatar then it depends on container block how to render the list of `Profile`(s)
Would that be possible?

Reply:
We can, though we would then be duplicating the layout styles as well as we are reusing them both in author-profiles and profiles. Perhaps we can have both profiles and profile as reusable classes?